syft/frameworks/torch/tensors/interpreters/native.py

```diff
@@ -246,7 +246,11 @@ def roll(tensor, shifts, **kwargs):
             int_shifts = int(shifts.item())
             return torch.native_roll(tensor, int_shifts, **kwargs)
 
+        def add(a, b):
+            return a + b
+
         module.roll = roll
+        module.add = add
```

Comment:
This will be only for the test, when you're happy with the PR remove these lines

Reply:
Ok!!